lib/_http_outgoing.js

```diff
@@ -424,6 +427,10 @@ function _storeHeader(firstLine, headers) {
         (state.contLen || this.useChunkedEncodingByDefault || this.agent);
     if (shouldSendKeepAlive) {
       header += 'Connection: keep-alive\r\n';
+      if (this._keepAliveTimeout) {
+        const timeoutSeconds = MathFloor(this._keepAliveTimeout) / 1000;
+        header += `Keep-Alive: timeout=${timeoutSeconds}\r\n`;
```

Comment:
Not add the header if the `Keep-Alive` exits maybe better.

Reply:
PR welcome

Reply:
https://github.com/nodejs/node/pull/35096